recipe-server/client/control_new/utils/apiFetch.js

```diff
@@ -16,7 +18,7 @@ export default async function apiFetch(url, options = {}) {
   if (!(options.body && options.body instanceof FormData)) {
     headers.append('Content-Type', 'application/json');
   }
-  headers.append('X-CSRFToken', document.getElementsByTagName('html')[0].dataset.csrf);
+  headers.append('X-CSRFToken', Cookie.get('csrftoken'));
```

Comment:
We should rename this to something like `csrftoken2` or `csrftoken-cachebust-2017-7-7` so that users don't have to clear their cookies manually when we deploy this.